### ISSUE TITLE
ext-pgsql - the requested PHP extension pgsql is missing from your system

### ISSUE THREAD

**Issue by eugenekkh** [opensoft/doctrine-postgres-types#13]
Hello. I try use dev-master version the package and it has troubles. I use Mac OS Snow Lion and php56 that installed through the brew. Composer tells me:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - opensoft/doctrine-postgres-types dev-master requires ext-pgsql * -> the requested PHP extension pgsql is missing from your system.
    - opensoft/doctrine-postgres-types dev-master requires ext-pgsql * -> the requested PHP extension pgsql is missing from your system.
    - Installation request for opensoft/doctrine-postgres-types dev-master -> satisfiable by opensoft/doctrine-postgres-types[dev-master].

  To enable extensions, verify that they are enabled in those .ini files:
    - /usr/local/etc/php/5.6/php.ini
    - /usr/local/etc/php/5.6/conf.d/ext-intl.ini
    - /usr/local/etc/php/5.6/conf.d/ext-mcrypt.ini
    - /usr/local/etc/php/5.6/conf.d/ext-memcache.ini
    - /usr/local/etc/php/5.6/conf.d/ext-pdo_pgsql.ini
  You can also run `php --ini` inside terminal to see which files are used by PHP in CLI mode.

```

Change ext-pgsql to ext-pdo_pgsql resolve the problem

**Comment by rybakit** [comment]
Neither `ext-pgsql` nor `ext-pdo_pgsql` requirement is needed for this package, https://github.com/opensoft/doctrine-postgres-types/pull/15 fixes this.

**Comment by richardfullmer** [comment]
Fixed by #15
